scipy/optimize/_highs/cython/src/_highs_wrapper.pyx

```diff
@@ -708,9 +701,8 @@ def _highs_wrapper(
             # Note: this is for all constraints (A_ub and A_eq)
             'slack': [rhs[ii] - solution.row_value[ii] for ii in range(numrow)],
 
-            # slacks in HiGHS appear as Ax - s, not Ax + s, so lambda is negated;
             # lambda are the lagrange multipliers associated with Ax=b
-            'lambda': [-1*solution.row_dual[ii] for ii in range(numrow)],
+            'lambda': [solution.row_dual[ii] for ii in range(numrow)],
```

Comment:
Sign convention flipped in recent version of HiGHS

Reply:
You noticed because tests were failing without this change? Is this change to HiGHS documented somewhere?

Reply:
No, this change was communicated a little while ago, see ERGO-Code/HiGHS#470